internal/pkg/model/object.go

```diff
@@ -88,7 +88,8 @@ type Config struct {
 	Description       string                 `json:"description" diff:"true" descriptionFile:"true"`
 	ChangeDescription string                 `json:"changeDescription"`
 	Content           *orderedmap.OrderedMap `json:"configuration" validate:"required" diff:"true" configFile:"true"`
-	Blocks            []*Block               `json:"-"` // loaded transformation's blocks, filled in only for the LOCAL state
+	Blocks            []*Block               `json:"-" validate:"dive"` // loaded transformation's blocks, filled in only for the LOCAL state
+	Relations         Relations              `json:"-" validate:"dive"`
```

Comment:
`Relations` pridane aj do `Config`, doteraz boli iba v `ConfigManifest`.

Vysvetlim to podrobnejsie.

Stav nejakeho objektu, v tomto pripade `Configu`, obsahuje strukura `ConfigState`:
https://github.com/keboola/keboola-as-code/blob/4a090b0fbb2af1c6b69f9ec8165ce03626891f65/internal/pkg/model/objectstate.go#L42-L46
- **`ConfigManifest`** je zaznam z `manifest.json`, bez zmeny, iba sa dopocita `parentPath`.
- **`Local *Config`** je lokalny stav, ... vznikne z `ConfigManifest` tak, ze sa nacitaju dalsie subory `meta.json`, `config.json` .... a spolu vytvoria obsah struktury `Config`. Do tohto procesu vstupuje `mapper`, ktorym je mozne vysledok ovplyvnit.
- **`Remote *Config`** je remote stav, .vznikne nacitanim objektu cez API, ... do tohto procesu vstupuje tiez `mapper`.

No a my sme mali `Relations` zatial iba v `ConfigManifest`, a v tomto PR sa dostanu do `Config`.
V niektorom z dalsi PR sa vyriesi aj `Remote` stav.